README.md

```diff
@@ -88,9 +88,9 @@ There are also aggregated json files with all chains automatically assembled:
 ## Collision management
 
  We cannot allow more than one chain with the same chainID - this would open the door to replay attacks.
- The first pull request gets the chainID assigned. When creating a chain we can expect that you read EIP155 which states this repo.
- All pull request trying to replace a chainID because they think their chain is better than the other will be closed.
- The only way to get a chain reassigned is when the old chain gets deprecated. This can e.g. be used for testnets that are short lived. But then you will get the redFlag "reusedChaiID" that should be displayed in clients to warn them about the dangers here.
+ The first pull requests gets the chainID assigned. When creating a chain we can expect that you read EIP155 which states this repo.
```

Comment:
I think this was correct - the rest are great fixes - but this one should stay IMHO

Reply:
Yup, sorry for that one, just changed!